docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+:code:`requirements.txt` files, then you may need to rebuild the JupyterLab application.
+
+For simplicity, we typically include the instructions for re-building JupyterLab in a 
+:code:`postBuild` script. Here is what this script looks like for my Horovod environments.
+
+.. code-block:: bash
+
+    jupyter labextension install --no-build jupyterlab-nvdashboard 
+    jupyter labextension install --no-build jupyterlab_tensorboard
+    jupyter lab build
+
+Use the following commands to source the :code:`postBuild` script.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX # optional if environment already active
+    $ . postBuild
+
+Listing the contents of the Conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To see the full list of packages installed into the environment run the following command.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX # optional if environment already active
+    $ conda list
+
+Verifying the Conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After building the Conda environment you can check that Horovod has been built with support for 
+the deep learning frameworks TensorFlow, PyTorch, Apache MXNet, and the contollers MPI and Gloo 
+with the following command.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX # optional if environment already active
+    $ horovodrun --check-build
+
+You should see output similar to the following.::
+
+    Horovod v0.19.4:
+    Available Frameworks:
+        [X] TensorFlow
+        [X] PyTorch
+        [X] MXNet
+    Available Controllers:
+        [X] MPI
+        [X] Gloo
+    Available Tensor Operations:
+        [X] NCCL
+        [ ] DDL
+        [ ] CCL
+        [X] MPI
+        [X] Gloo
+
+Wrapping it all up in a Bash script
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We typically wrap these commands into a shell script :code:`create-conda-env.sh`. Running the shell 
+script will set the Horovod build variables, create the Conda environment, activate the Conda 
+environment, and built JupyterLab with any additional extensions.
+
+.. code-block:: bash
+
+    #!/bin/bash --login
+
+    set -e
+    
+    export ENV_PREFIX=$PWD/env
+    export HOROVOD_CUDA_HOME=$CUDA_HOME
+    export HOROVOD_NCCL_HOME=$ENV_PREFIX
+    export HOROVOD_GPU_OPERATIONS=NCCL
+    conda env create --prefix $ENV_PREFIX --file environment.yml --force
+    conda activate $ENV_PREFIX
+    . postBuild
+
+We recommend that you put scripts inside a :code:`bin` directory in your project root directory. 
+The script should be run from the project root directory as follows.
+
+.. code-block:: bash
+
+    ./bin/create-conda-env.sh # assumes that $CUDA_HOME is set properly
+
+Updating the Conda environment
+------------------------------
+
+If you add (remove) dependencies to (from) the :code:`environment.yml` file or the 
+:code:`requirements.txt` file after the environment has already been created, then you can 
```

Comment:
I recommend using double backquotes (``) for code highlighting

Reply:
Done